ruby/ql/src/queries/security/cwe-598/SensitiveGetQuery.qhelp

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+<overview>
+<p>
+Sensitive information such as user passwords should not be transmitted within the query string of the requested URL.
```

Comment:
Did you mean `password` or `user names and passwords`. While `user passwords` is a correct term, I think transmitting non-user passwords would be equally problematic.

Reply:
Yeah, just `passwords` makes more sense here. We don't report user names here as they're not considered sensitive in this context.